Utils/DoCommand/DoCommand.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <unistd.h>
+#include <sys/wait.h>
+
+void DoCommand(const std::string& cmd){
+    auto child = fork();
+    if (child == 0)
+    {
+        execl("/bin/sh","sh" ,"-c", cmd.c_str(), nullptr);
+    }
+    else
+    {
+        int ret;
+        wait(&ret);
```

Comment:
Use waitpid. The wait system call waits until any child process becomes a zombie. Waitpid waits for a specific child process.